openmetadata-service/src/main/java/org/openmetadata/service/resources/elasticSearch/BuildSearchIndexResource.java

```diff
@@ -237,23 +219,32 @@ private synchronized void submitBatchJob(UriInfo uriInfo, String startedBy, Crea
 
     // Update Listener for only Batch
     BulkProcessorListener bulkProcessorListener = new BulkProcessorListener(dao);
-    BulkProcessor processor = getBulkProcessor(bulkProcessorListener);
+    BulkProcessor processor =
+        getBulkProcessor(bulkProcessorListener, createRequest.getBatchSize(), createRequest.getFlushIntervalInSec());
 
-    if (createRequest.getEntities().contains("all")) {
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, TABLE, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, TOPIC, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, DASHBOARD, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, PIPELINE, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, MLMODEL, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, USER, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, TEAM, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, GLOSSARY_TERM, createRequest);
-      updateEntityBatch(processor, bulkProcessorListener, uriInfo, TAG, createRequest);
-    } else {
-      for (String entityName : createRequest.getEntities()) {
-        updateEntityBatch(processor, bulkProcessorListener, uriInfo, entityName, createRequest);
-      }
+    for (String entityName : createRequest.getEntities()) {
```

Comment:
What happens if "all" is still passed in `entities`?

Reply:
it was never used, from the UI itself we are always passing the specific entities